.github/dependabot.yml

```
@@ -0,0 +1,11 @@
+---
+# Copyright (c) Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
```

How about quarterly?

---

What is `quarterly`?

---

I'd actually go for `day: "tuesday"` as a personal preference. weekly feels like the right frequency imo though.

---

Oh, dependabot doesn't have that. I confused it with pre-commit.ci.
So at least do
```suggestion
      interval: monthly
```

---

monthly seems too infrequent. weekly is a good comprise, IMO.

---

It is too noisy if you get notifications from a lot of projects. And there's no easy way to turn off just that noise on the scale…